Formula/protobuf.rb

```diff
@@ -26,11 +27,7 @@ class Protobuf < Formula
   end
 
   depends_on "python@3.9" => [:build, :test]
-
-  resource "six" do
-    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
-    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
-  end
+  depends_on "six"
```

Comment:
How can it use six at runtime without python?

Reply:
Presumably it uses six when you use the Python bindings. Otherwise the existing resource wouldn't need to be installed into `libexec`.

Reply:
Maybe I'm missing something, but why does this change whether `python@3.9` is a runtime dependency or not?

Reply:
Adding a `depends_on "six"` will require installing `python@3.9` when you install `protobuf` (because `six` `depends_on "python@3.9"`). Without this change, you can install `protobuf` without `python@3.9` installed.

Reply:
Oh, duh 🤦